test/e2e/extended/cat-b-scenarios.feature

```diff
@@ -0,0 +1,198 @@
+      And I proceed to the car
+      Then I should see the "Florence Pearson" page
+      And I complete the waiting room to car page
+      Then I should see the "Test report - Florence Pearson" page
+      When I add a "Accelerator" driver fault
+      And I add a "Timed" driver fault
+      And I add a "Clearance" driver fault
+      And I add a "Signalling" driver fault
+      And I add a "Timed" driver fault
+      Then the driver fault count is "5"
+      And I complete the test
+      And I continue to debrief
+      Then I should see the Debrief page with outcome "Passed"
+      When I end the debrief
+      Then I should see the "Test debrief - Florence Pearson" page
+      And I complete the pass details
+      And I complete the health declaration
+      Then I am on the back to office page
+      And I continue to the office write up
+      Then I should see the "Office" page
+      And I complete the office write up
+      And I upload the test
+      Then I should see the "Journal" page
+
+   Scenario: Candidate fails a test with a single dangerous fault
+      Given I am logged in as "mobexaminer1" and I have a test for "Mr Ali Campbell"
+      When I start the test for "Mr Ali Campbell"
+      And the candidate requests to receive results by post
+      And the candidate confirms their communication preference
+      Then I should see the "Declaration - Ali Campbell" page
+      And the candidate completes the declaration page
+      And I proceed to the car
+      Then I should see the "Ali Campbell" page
+      And I complete the waiting room to car page
+      Then I should see the "Test report - Ali Campbell" page
+      When I add a "Use of speed" dangerous fault
+      And I add a "Safety" driver fault
+      And I add a "Lane discipline" driver fault
+      When I complete the test
+      And I continue to debrief
+      Then I should see the Debrief page with outcome "Unsuccessful"
+      When I end the debrief
+      Then I am on the back to office page
+      And I continue to the office write up
+      Then I should see the "Office" page
+      And I complete the office write up
+      And I enter a comment for "dangerous" fault "Use of speed"
+      And I upload the test
+      Then I should see the "Journal" page
+
+   Scenario: Candidate fails a test with a single serious fault
+      When I start the test for "Mrs Jane Doe"
+      And the candidate confirms their communication preference
+      Then I should see the "Declaration - Jane Doe" page
+      And the candidate completes the declaration page
+      And I proceed to the car
+      Then I should see the "Jane Doe" page
+      And I complete the waiting room to car page with a tell me driver fault
+      Then I should see the "Test report - Jane Doe" page
+      When I add a "Accelerator" serious fault
+      When I complete the test
+      And I continue to debrief
+      Then I should see the Debrief page with outcome "Unsuccessful"
```

Comment:
Maybe some assertion about the list of faults? I see we have this for the full journey suite, so would have thought they'd be included in extended suite.